lib/_debugger.js

```diff
@@ -1737,7 +1738,7 @@ Interface.prototype.trySpawn = function(cb) {
   function connectError() {
     // If it's failed to connect 10 times then print failed message
     if (connectionAttempts >= 10) {
-      console.error(' failed, please retry');
+      internalUtil.error(' failed to connect, please retry');
```

Comment:
Not sure about the context of this error, but the message change seems unnecessary.